lib/rules/must-tear-down-animations.js

```diff
@@ -19,13 +15,15 @@ const create = (context) => {
      */
     function reportMissingTeardowns() {
         // Get list of all animated nodes that remain active (not torn down)
-        const activeNodes = Object.values(activeAnimations);
+        const componentAnimations = Object.values(activeAnimations);
         // Iterate through all nodes failing the lint rule
         // and use context.report to surface error to user.
-        activeNodes.forEach(node => {
-            context.report({
-                node: node,
-                message: `Must tear down animations when component unmounts`,
+        componentAnimations.forEach(activeNodes => {
+            Object.values(activeNodes).forEach(node => {
+                context.report({
+                    node: node,
```

Comment:
I guess the node contains location info and eslint probably has a standard way of including that information when it logs an error/warning message.